src/Services/Basket/Basket.UnitTests/Basket.UnitTests.csproj

```diff
@@ -7,7 +7,11 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore.Hosting" Version="2.2.0" />
```

Comment:
Why include references that are not needed?

Reply:
removed!